integration/kubernetes/confidential/agent_image.bats

```diff
@@ -126,28 +132,45 @@ setup() {
 	echo $pod_config
 
 	create_test_pod
+	kubectl delete secret cococred --ignore-not-found
 }
 
 @test "$test_tag Test cannot pull an image from an authenticated registry with incorrect credentials" {
 	if [ "${AA_KBC}" = "cc_kbc" ]; then
 		skip "As the test requires changing verdictd configuration and restarting its service"
 	fi
 
+	kubectl delete secret cococred --ignore-not-found
+	kubectl create secret docker-registry cococred --docker-server="https://quay.io/kata-containers/confidential-containers-auth" \
+	--docker-username="Arandomquaytestaccountthatdoesntexist" --docker-password="password"
+
 	REGISTRY_CREDENTIAL_ENCODED="QXJhbmRvbXF1YXl0ZXN0YWNjb3VudHRoYXRkb2VzbnRleGlzdDpwYXNzd29yZAo=" setup_credentials_files "quay.io/kata-containers/confidential-containers-auth"
 
 	pod_config="$(new_pod_config "${image_authenticated}")"
 	echo "Pod config: ${pod_config}"
 
 	assert_pod_fail "${pod_config}"
-	assert_logs_contain 'failed to pull manifest Authentication failure'
+	assert_logs_contain "containerd" 'failed to fetch oauth token'
+	kubectl delete secret cococred --ignore-not-found
 }
 
 @test "$test_tag Test cannot pull an image from an authenticated registry without credentials" {
 	pod_config="$(new_pod_config "${image_authenticated}")"
 	echo "Pod config: ${pod_config}"
 
 	assert_pod_fail "${pod_config}"
-	assert_logs_contain 'failed to pull manifest Not authorized'
+
+	# Print the logs 
+	echo "-- Kata logs:"
+	sudo journalctl -xe -t kata --since "$test_start_date" -n 100000
+
+	echo "-- containerd logs:"
+	sudo journalctl -xe -t containerd --since "$test_start_date" -n 100000
+
+	echo "-- kubelet logs:"
+	sudo journalctl -xe -t kubelet --since "$test_start_date" -n 100000
```

Comment:
I think we can probable delete these debug logs (or move them into the teardown if they are useful)